Sources/Apodini/Semantic Model Builder/Model/Endpoint.swift

```diff
@@ -64,26 +64,27 @@ struct Endpoint<H: Handler>: AnyEndpoint {
     
     /// All `@Parameter` `RequestInjectable`s that are used inside handling `Component`
     var parameters: [AnyEndpointParameter]
-    
-    var absolutePath: [_PathComponent] {
-        treeNode.absolutePath
+
+    var absolutePath: [EndpointPath] {
+        storedAbsolutePath
     }
+    private var storedAbsolutePath: [EndpointPath]! // swiftlint:disable:this implicitly_unwrapped_optional
+
     var relationships: [EndpointRelationship] {
-        treeNode.relationships
+        storedRelationship
     }
+    private var storedRelationship: [EndpointRelationship]! // swiftlint:disable:this implicitly_unwrapped_optional
```

Comment:
I don't really see the need to basically declare the same attribute twice. I think this solution would also work and achieve the same thing:

```swift
private(set) absoulePath: [EndpointPath] = []
private(set) relationships: [EndpointRelationship] = []
```
This would also remove the need for force unwrapping

Reply:
The intention here is, that both properties are not considered valid until inserted into a `EndpointTreeNode`. If one would access those properties before it got inserted, you know something has gone fatally wrong.
I wanted to make that explicit in the code, and that's the reason I decided against doing an empty default value as you suggest it.
I could find a way to remove the implicitly unwrapped optional (by making the `stored...` properties Optional and doing an unwrap in the computed properties, creating a fatalError if they are indeed nil.
However I'm not sure how to achieve the same thing with removing the additional properties 🤔